chore(docs): clean up and add new content (#15003)
immich-app/immich: docs/docs/overview/quick-start.mdx

You can select the **Jobs** tab to see Immich processing your photos.

<img src={require('/docs/guides/img/jobs-tab.png').default} title="Jobs tab" width={300} />

> ## Familiar with database backup and restore process

Reviewer: ```suggestion
## Review the database backup and restore process
```

> Immich has a built-in database backup and restore feature. You can refer to the

Reviewer: ```suggestion
Immich has built-in database backups. You can refer to the
```

[database backup](/docs/administration/backup-and-restore) for more information.

> 

Reviewer: ```suggestion
:::danger
The database only contains metadata and user information. You must setup manual backups of the images and videos stored in `UPLOAD_LOCATION`.
:::
```

[upload photos from your own archive](/docs/features/command-line-interface).

> You may want to incorporate an immutable archive of photos from an

Reviewer: ```suggestion
You may want to incorporate a pre-existing archive of photos from an
```

[External Library](/docs/features/libraries);

> there's a [Guide](/docs/guides/external-library) for that.

Reviewer: ```suggestion
there's a [guide](/docs/guides/external-library) for that.
```
